Terraform/terraform-deployment.yml

```diff
@@ -0,0 +1,61 @@
+name: "Terraform Deploy"
+on:
+  push:
+    branches:
+      - master
+
+env:
+  TF_VER: 0.12.29
+  TF_WORKING_DIR: "terraform_templates"
```

Comment:
This puts a ask on users to have "terraform_templates" directory in their repo. Should we instead start with current directory (".") as the working directory?

@azooinmyluggage / @N-Usha ?

Reply:
Agreed. Again Would like @manojvazirani to clarify if the rationale behind this. Thanks!

Reply:
Thanks for your review. I checked that if working directory is not mentioned then it would take the root directory as default. Removed the requirement for working directory.